vendors/marvell/WMSDK/mw320/sdk/external/utils/build.mk

```diff
@@ -5,7 +5,7 @@ libs-y += libutils
 
 libutils-objs-y := \
 	           ../../../../../../../libraries/freertos_plus/standard/utils/src/iot_system_init.c \
-		   ../../../../../../../libraries/3rdparty/mbedtls/utils/mbedtls_utils.c\
+		   ../../../../../../../libraries/3rdparty/mbedtls_utils/mbedtls_utils.c\
```

Comment:
My understanding is also that Marvell builds with cmake. I'm not sure why there are makefiles here. I noticed that it would need to be updated when I was searching for places that referenced mbedtls_utils.c. I'll look into this and get back to you.

Reply:
Tiangang, you are right, in our CI we are only using CMake for Marvell projects. So was the case during qualification. 

The story for Marvell having build.mk in SDK was... they submitted Eclipse project files (for viewing only) and CMake projects. The build.mk files in SDK are likely for building projects with make when CMake wasn't in place, that the path referenced are based on old directory structure as you may have recognized. In the middle of everything happened last year, we went straight with CMake, and .mk files in SDK were left as-is. 

Going forward I'd propose below: 
- Leave out this change in this PR, and confirm Marvell port still works. 
- Marvell Makefile story is very similar to ESP Makefile story. If we have done something different for ESP, we could follow the same approach for Marvell.

Reply:
@yuhui-zheng Thanks for the clarification. I agree to leave out the makefile change in this PR. For ESP32 because we still support building with makefile so that it make sense to maintain them. However for marvell we don't support makefile in the beginning nor we have such plan, so it's better to delete them to avoid confusion.

Reply:
@tgsong I agree with keeping the changes to the espressif project since it's actively maintained/supported. 

[I reverted the change to the Marvell makefile. ](https://github.com/aws/amazon-freertos/pull/1881/commits/bdf9345bd497cb300c49176c169acf341ef4a7d4)